extract/sbs_crawler.py

```
@@ -0,0 +1,100 @@
+        }
+        print(params)
+        page_response = requests.get(self.page_base_url, params=params)
+        if page_response.status_code == 200:
+            news_list = page_response.json()['news_sbs']
+        return news_list
+
+    def get_news_content(self, news_list:List[dict])->List[NewsResponse]:
+        news_output:List[NewsResponse] = []
+        for news in news_list:
+            news_title = news['TITLE']
+            news_summary = news['REDUCE_CONTENTS']
```

Ruff (0.8.2)

Local variable `news_summary` is assigned to but never used

Remove assignment to unused variable `news_summary`

(F841)